newrelic/resource_newrelic_synthetics_monitor_test.go

```diff
@@ -18,6 +19,97 @@ import (
 
 var tv bool = true
 
+func TestAccNewRelicSyntheticsBrowserMonitor_DeviceEmulationError(t *testing.T) {
```

Comment:
Can you add a similar test case to test if the error is seen when an existing Terraform configuration is updated?